### PR TITLE
add extra translations, make corrections

### PR DESCRIPTION
**Pull request by samundra**
**add translations on**
- disciple
- encouragement

**corrections made on**
- epic
- fade
- illustrate
- sincere
- sing

### CODE REVIEW

File: en2np/main.tra [hunk 1]
```diff
@@ -1038,6 +1039,7 @@ emulation; यंत्रानुकरण
 enable; [इनेबल] v(अधिकार दिनु, प्राप्त गर्नु, सक्षम बनाउनु, लागु गर्नु, सुचारु गर्नु), ~on
 encoding; [इन्कोडिङ] सङ्केतन
 encourage; v(प्रोत्साहित)
+encouragement; n(प्राेत्साहन)
```
Comment (rhoit) [inline en2np/main.tra:1042]: you wrote
प्राे ‍→ प ‌+ ् + र ‌+ ा + े

but should be 
प्रो → प + ् + र ‌+ ो

try [m17n-ne](https://github.com/rhoit/m17n-ne) lipi for such auto correction

File: en2np/main.tra [hunk 2]
```diff
@@ -2534,6 +2536,7 @@ shout; [शाउट्] v(कराउनु, चिच्याउनु), v(
 show; दर्शाउँछ, प्रदर्शन, चर्तिकला
 shrimp; झिंगे माछा #animal
 shutdown; बन्द गर्नुहोस
+sick; j(बिमार, राेगी) @ill
```
Comment (rhoit) [inline en2np/main.tra:2539]: same ा + े problem

**Comment by samundra** [comment]
I used the windows version of nepali layout that comes default with windows8. It has layout changes like you mentioend. Can I use the one promoted by Madan Puruskar, have you used that one before?

**Comment by rhoit** [comment]
its just a suggestion for linux no idea about windows 8, MPP made that layout 10 years ago, its time to change.

**Comment by samundra** [comment]
That's fine. I have been looking for keyboard layout that you've mentioned. I will try them in linux. Since I have to keep switching between linux and windows, I am looking for alternative to windows current keyboard layout. I agree, the keyboard layout is 10 years old now, I have gotten so used to it that I have forgotten to type in Pretti :stuck_out_tongue:

**Comment by rhoit** [comment]
@samundra may be some day we will port it to windows but for now its different story!

**Comment by samundra** [comment]
Eagerly waiting for the day. For all future PR I will use the one you suggested. Thanks.

**Pull request merged**